main.py

```diff
@@ -1,7 +1,50 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        ("Western",),
```

Comment:
here can be a list of strings or tuple of strings. No need to create a list of tuples of strings  :)